flink-runtime/src/test/java/org/apache/flink/runtime/io/network/partition/consumer/LocalInputChannelTest.java

```diff
@@ -114,11 +114,12 @@ public void testConcurrentConsumeMultiplePartitions() throws Exception {
 				.setNumTargetKeyGroups(parallelism)
 				.setResultPartitionManager(partitionManager)
 				.setSendScheduleOrUpdateConsumersMessage(true)
+				.setBufferPoolFactory(p ->
+					networkBuffers.createBufferPool(producerBufferPoolSize, producerBufferPoolSize))
 				.build();
 
 			// Create a buffer pool for this partition
```

Comment:
comment seems a bit out-dated, I'd probably just remove it

Reply:
well, this is still what actually happens atm, why is it out-dated?